### PR TITLE
Tested locally without frontend

### PR DESCRIPTION
**Pull request by samimshoaib01**
When we change the : Available projects in .env file , it get reflects in /project 
Front end is not maintained as before 

Here we changed AVAILABLE PROJECTS in .env file

Our /project is changed

Also in installation guide it would be great to mention that Contributor should use **python 3.9** as python 3 (default one) makes it  complex to set up in linux. I spent bunch of hours fixing that.

### CODE REVIEW

**Comment by samimshoaib01** [comment]
Update the frontend: 
Screenshot from 2024-10-20 04-33-13
Functionality to redirect to repo :

https://github.com/user-attachments/assets/33332256-fcdf-4d58-b7ad-48b0a74272b7


Regarding maintainer name:

 {
    "id": 144955661,
    "node_id": "MDEwOlJlcG9zaXRvcnkxNDQ5NTU2NjE=",
    "name": "ContriHub-18",
    "full_name": "ContriHUB/ContriHub-18",
    "private": false,
    "owner": {
      "login": "ContriHUB",
      "id": 42784838,
      "node_id": "MDEyOk9yZ2FuaXphdGlvbjQyNzg0ODM4",
      "avatar_url": "https://avatars.githubusercontent.com/u/42784838?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/ContriHUB",
      "html_url": "https://github.com/ContriHUB",
      "followers_url": "https://api.github.com/users/ContriHUB/followers",
      "following_url": "https://api.github.com/users/ContriHUB/following{/other_user}",
      "gists_url": "https://api.github.com/users/ContriHUB/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ContriHUB/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ContriHUB/subscriptions",
      "organizations_url": "https://api.github.com/users/ContriHUB/orgs",
      "repos_url": "https://api.github.com/users/ContriHUB/repos",
      "events_url": "https://api.github.com/users/ContriHUB/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ContriHUB/received_events",
      "type": "Organization",
      "user_view_type": "public",
      "site_admin": false
    },
I think owner=>login should be the name of the maintainer , but every where it is ContriHUB .

File: contrihub/settings.py [hunk 1]
```diff
@@ -185,7 +185,7 @@
 EMAIL_USE_SSL = config('EMAIL_USE_SSL', default=False, cast=bool)
 
 AVAILABLE_PROJECTS = config('AVAILABLE_PROJECTS', default="ContriHUB-24",
-                            cast=lambda v: [s.strip() for s in v.split(',')])
```
Comment (meisabhishekpatel) [inline contrihub/settings.py:188]: leave it to "," only as delimiter

File: project/views.py [hunk 2]
```diff
@@ -21,12 +68,10 @@ def populate_projects(request):
     """
     Used to Populate Projects in Local Database. Creates entries based on project names present in AVAILABLE_PROJECTS
     config variable in settings.py
-    :param request:
-    :return:
     """
```
Comment (meisabhishekpatel) [inline project/views.py:71]: why are you deleting the comments (parameter and return type) and this is decreasing the code quality.

File: project/views.py [hunk 3]
```diff
@@ -65,8 +108,6 @@ def populate_issues(request):
         if response['status'] == SUCCESS:
             issues = response['data']
             for issue in issues:
-                # print(issue)
-                # TODO: Can be given as ISSUE
                 try:
```
Comment (meisabhishekpatel) [inline project/views.py:111]: here as well this code was added for future reference

File: project/views.py [hunk 4]
```diff
@@ -75,12 +116,11 @@ def populate_issues(request):
                 finally:
                     print('error')
                 if issue.get('pull_request') is not None:  # this issue is actually a PR.
-                    # Source: https://docs.github.com/en/rest/reference/issues#list-repository-issues
-                    print("This issue is a actually a PR")
+                    print("This issue is actually a PR")
```
Comment (meisabhishekpatel) [inline project/views.py:119]: here also why did you remove the source for documentation

File: project/views.py [hunk 5]
```diff
@@ -128,11 +167,10 @@ def parse_labels(labels):
     for label in labels:
 
         if str(label["description"]).lower() == LABEL_MENTOR:  # Parsing Mentor
-            # print("dfs ",label["name"])
             mentor = parse_mentor(label["name"])
 
         if str(label["description"]).lower() == LABEL_LEVEL:  # Parsing Level
-            level, points = parse_level(label["name"])  # Fetching Level and it's default point
+            level, points = parse_level(label["name"])  # Fetching Level and its default point
```
Comment (meisabhishekpatel) [inline project/views.py:173]: it's its which is correct ?

File: project/views.py [hunk 6]
```diff
@@ -46,8 +91,6 @@ def populate_projects(request):
 def populate_issues(request):
     """
     Used to Populate Issues in Local Database. It fetches Issues from Github using Github API.
-    :param request:
-    :return:
     """
```
Comment (meisabhishekpatel) [inline project/views.py:94]: here too.

**Comment by samimshoaib01** [comment]
Kindly check sir

File: project/views.py (hunk 5 above)
Comment (samimshoaib01) [reply project/views.py:183]: Sir I asked GPT it says its will be correct

**Comment by samimshoaib01** [comment]
@meisabhishekpatel Is there any issue sir  ?

**Comment by meisabhishekpatel** [comment]
closing this issue.
